lib/widgets/selection.dart

```diff
@@ -1058,32 +1069,26 @@ class _ActionsSelectionTitle extends StatelessWidget {
           ),
         if (selectedTitle && counter)
           Padding(
-            padding: EdgeInsets.only(bottom: 2.0, left: closeButton ? 12.0 : 30.0),
+            padding: EdgeInsets.only(bottom: 4.0, left: closeButton ? 12.0 : 30.0),
```

Comment:
Actually, the 4 pixel padding is required to keep the same vertical position as before. It is a combination of the removed 2 pixel in https://github.com/nt4f04uNd/sweyer/pull/24/files#diff-f0830ada064506aee2ff71a6a8a16544cec614957b4c7307f28177163ea2dbc7L1072-L1074 and the previous 2. We have to do this because the counter is inside of the text widget now.

| After - With only 2px | Before | After - Current (4px) |
|------------------------|---------|-----------------------|
| ![Aftet 2](https://user-images.githubusercontent.com/6966049/168318453-6271c407-f4e8-4615-a22f-f2f8933025a5.png) | ![Before](https://user-images.githubusercontent.com/6966049/168318466-2b8aa94d-4201-4a88-a435-4eac8761c2e0.png) | ![Aftet 4](https://user-images.githubusercontent.com/6966049/168318479-24518537-95a3-430a-a7db-267371c6a66a.png) |

Reply:
Oh, ok fine